andela/ah-frontend-athena
# #162163222 update TravisCI to run tests

#### What does this PR do?
Add travis continous Intergration to the repository

#### Description of Task to be completed?
- travis build is triggered whenever an action is performed on the repo
- add travis and codeclimate badges

#### How should this be manually tested?
Push your branch to the repository, the travis badge should be passing and a lighting green. Click on the badge to view your running tests.

#### Any background context you want to provide?
Enable Continous Intergration.

#### What are the relevant pivotal tracker stories?
[#162163222](https://www.pivotaltracker.com/story/show/162163222)

#### Screenshots
<img width="505" alt="screen shot 2019-01-09 at 14 19 29" src="https://user-images.githubusercontent.com/32167860/50896279-a49cfb00-1419-11e9-9ec0-0ef8388f2766.png">
